arc.go

```diff
@@ -416,3 +416,69 @@ func (al *arcList) RemoveTail() interface{} {
 func (al *arcList) Len() int {
 	return al.l.Len()
 }
+
+//return the ttl of the item
+func (it *arcItem) GetExpired(now *time.Time) (bool, time.Duration) {
```

Comment:
I see how you are using `GetExpired` but I don't understand why you are making it an exported function. If you do choose to go down this way (see my other comment) it should be `getExpired` IMO.